### ISSUE TITLE
Best Algorithm Class inherited from DataSet class

### ISSUE THREAD

**Issue by GoogleCodeExporter** [numbbo/coco#46]
```
implement a (new) best algorithm class that derives from the DataSet class

- the DataSet class should become a new-style class (inheriting from object) 
first!
- implementing a (new) best algorithm class that derives from the DataSet class 
looks simple: only for each target the evals-line with the best ERT or best 
median f-value (or best f-value?) needs to be inserted. 

related to #45


```

Original issue reported on code.google.com by `dimo.bro...@inria.fr` on 16 Jun 2014 at 12:22

**Comment by nikohansen** [comment]
See also #19 #20 #45 #56

**Comment by brockho** [comment]
Can also be closed (the actual commit was e5d9eceefa9e9673002e96005138adb1aac94d36 from Oct. 12, 2016).
